lldb/source/Plugins/Language/Swift/SwiftOptionSet.cpp

```diff
@@ -174,6 +174,14 @@ bool lldb_private::formatters::swift::SwiftOptionSetSummaryProvider::
       if ((case_value & value) == case_value) {
         // hey a case matched!!
         any_match = true;
+
+        // Print single valued sets without enclosing in brackets.
+        if (first_match && case_value == value) {
+          ss.Printf(".%s", val_name.second.AsCString());
```

Comment:
Does StreamString have an `operator<<()`? Then we could avoid the C string conversion.

Reply:
It does, thanks.